### ISSUE TITLE
Accessibility Please

### ISSUE THREAD

**Issue by bkardell** [ebidel/polymer-experiments#6]
Keyboard and screen reader support is lacking I guess - I didn't try or look, but Marcy says so, so I believe it :)

https://twitter.com/marcysutton/status/537310830128160768
